<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>711</width>
    <height>547</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QGroupBox" name="grpBox_login">
           <property name="title">
            <string>Login</string>
           </property>
           <layout class="QFormLayout" name="formLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="lblusername">
              <property name="text">
               <string>User Name</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLineEdit" name="lnEusername">
              <property name="placeholderText">
               <string>who are you?</string>
              </property>
              <property name="clearButtonEnabled">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QGroupBox" name="grpBox_op_sel">
           <property name="title">
            <string>Operation Selection</string>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QRadioButton" name="rdbtn_roi">
                <property name="text">
                 <string>Draw ROI (polygon/rectangle)</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="rdbtn_edit">
                <property name="text">
                 <string>Review and Edit </string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="chBox_save_result_images">
                <property name="text">
                 <string>Save output images</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QGroupBox" name="grpBox_path">
           <property name="title">
            <string>Path Selection</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QLineEdit" name="lnE_load_img">
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
                <property name="dragEnabled">
                 <bool>false</bool>
                </property>
                <property name="readOnly">
                 <bool>false</bool>
                </property>
                <property name="placeholderText">
                 <string>Input images directory</string>
                </property>
                <property name="clearButtonEnabled">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lnE_load_text">
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
                <property name="dragEnabled">
                 <bool>false</bool>
                </property>
                <property name="readOnly">
                 <bool>false</bool>
                </property>
                <property name="placeholderText">
                 <string>input text file path</string>
                </property>
                <property name="clearButtonEnabled">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lnE_store_img">
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
                <property name="dragEnabled">
                 <bool>false</bool>
                </property>
                <property name="readOnly">
                 <bool>false</bool>
                </property>
                <property name="placeholderText">
                 <string>store images directory</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lnE_store_text">
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
                <property name="dragEnabled">
                 <bool>false</bool>
                </property>
                <property name="readOnly">
                 <bool>false</bool>
                </property>
                <property name="placeholderText">
                 <string>store text file path</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="0" column="1">
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QPushButton" name="btn_load_img">
                <property name="text">
                 <string>Load Image</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btn_load_text">
                <property name="text">
                 <string>Load Text</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btn_store_img">
                <property name="text">
                 <string>Store Image</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btn_store_text">
                <property name="text">
                 <string>Store Text</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="btn_enter">
         <property name="text">
          <string>Enter</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>711</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Help">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
